docs/branch-strategy.md

```
@@ -0,0 +1,503 @@
+git checkout -b uat
+git push origin -u uat
+
+# Create Dev from production
+git checkout production
+git checkout -b dev
+git push origin -u dev
+```
+
+## Visual Flow Diagram
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)